src/components/RadioTile/RadioTile.tsx

```diff
@@ -0,0 +1,94 @@
+import * as RadixRadioGroup from "@radix-ui/react-radio-group";
+import { Box, sprinkles, Text } from "@saleor/macaw-ui-next";
+import React from "react";
+
+import { getBgColor, getBorderColor, getHoverState } from "./utils";
+
+export interface RadioTileProps {
+  checked: boolean;
+  title: string;
+  description: string;
+  value: string;
+}
+
+export const RadioTile = ({
+  checked,
+  title,
+  description,
+  value,
+}: RadioTileProps) => {
+  const [isHoverState, setHoverState] = React.useState(false);
+
+  return (
+    <RadixRadioGroup.Item value={value} asChild>
```

Comment:
question: how could I know that I have to wrapp this component into `<RadixRadioGroup.Root>`, Would it be better to create `RadioTiles` component that will be wrapping all children and `RadioTile` will be it command component like this

```
const RadioTilesBase = ({children}) => {
    return (
        <RadixRadioGroup.Root>
            {children}
      </RadixRadioGroup.Root>,
    )
}

export const RadioTiles = Object.assign(RadioTilesBase, {
    RadioTile
})
```

Reply:
That's a fair point. I'll update this component